packages/batcher/webserver/src/index.ts

```diff
@@ -451,6 +460,45 @@ async function initializeServer(
   );
 }
 
+//
+// This method executes Google reCaptcha3 validation for backend.
+// If the validation fails, an error is thrown - otherwise, the method returns void.
+// To enable validation:
```

Comment:
This is the kind of stuff we should mention in the Paima docs. I don't really mind repeating it here, but it will be duplicate comments. I tend to prefer putting it in the docs and then linking to the docs from the docs (it does mean the link in the code breaks if the doc url changes, but usually this isn't that big of a deal imo since knowing it's somewhere in the docs is usually enough)